installer/templates/phx_assets/bare/gitignore

```diff
@@ -1,12 +1,30 @@
-# App artifacts
-/_build
-/db
-/deps
-/*.ez
+# The directory Mix will write compiled artifacts to.
+/_build/
 
-# Generated on crash by the VM
+# If you run "mix test --cover", coverage assets end up here.
+/cover/
+
+# The directory Mix downloads your dependencies sources to.
+/deps/
+
+# Where 3rd-party dependencies like ExDoc output generated docs.
+/doc/
+
+# Ignore .fetch files in case you like to edit your project deps locally.
+/.fetch
+
+# If the VM crashes, it generates a dump, let's ignore it too.
 erl_crash.dump
 
+# Also ignore archive artifacts (built via "mix archive.build").
+*.ez
+
+# Ignore package tarball (built via "mix hex.build").
+<%= app_name %>-*.tar
+
+# Database data
```

Comment:
What database is this for?  I know it's a common Mnesia directory, but this doesn't say that?

Reply:
To be honest, I saw it in the previous template but I was not sure in which scenarios that directory was generated.

Glad to help the description to something better :) Thanks!

Reply:
sqlite3

Reply:
Changed to:

```
# Ignore the SQLite 3 database files.
/db/*.sqlite3
/db/*.sqlite3-journal
```

Done in https://github.com/phoenixframework/phoenix/pull/2772/commits/e111611753bbc47f955aca45632e1c08da71280e

Reply:
On the other hand, [the generator does not include `sqlite` in the list of Ecto adapters](https://github.com/phoenixframework/phoenix/blob/master/installer/lib/phx_new/generator.ex#L178-L189), should we then remove the `/db` directory from the template?

The line `/db` was introduced in the previous template [in this commit](https://github.com/phoenixframework/phoenix/commit/f0f8c5341f485ffbd1222dda06d02a4c552664ea). However, support for SQLite was removed in [this commit](https://github.com/phoenixframework/phoenix/commit/004ed0bd7618f297569a5abc0abc487ae215b1a2#diff-62c3c1595e6fe80fd313b78665b1f44d).

Reply:
Removed in https://github.com/phoenixframework/phoenix/pull/2772/commits/01caf31d67f88bcda77470adf6e877f4a6856e0e